fem/bilininteg.cpp

```diff
@@ -600,17 +664,17 @@ void MixedScalarVectorIntegrator::AssembleElementMatrix2(
    int trial_nd = trial_fe.GetDof(), test_nd = test_fe.GetDof(), i;
    int sca_nd = sca_fe->GetDof();
    int vec_nd = vec_fe->GetDof();
-   int spaceDim = Trans.GetSpaceDim();
+   int vdim = GetVDim(*vec_fe);
```

Comment:
This is a tricky one. In this case `GetVDim` is a member function of the integrator which is meant to be overridden by derived classes. So, we don't want to supersede choices made in the derived classes. Perhaps the best resolution is to store the space dimension as member data so that we can compute the `max` inside `GetVDim` by default but still allow derived classes to make their own determination.